lib/components/narrative/default/default-itinerary.js

```diff
@@ -1,34 +1,76 @@
+import moment from 'moment-timezone'
 import coreUtils from '@opentripplanner/core-utils'
 import React from 'react'
+import { FormattedMessage, FormattedNumber } from 'react-intl'
+import { connect } from 'react-redux'
 
 import NarrativeItinerary from '../narrative-itinerary'
 import ItineraryBody from '../line-itin/connected-itinerary-body'
-import ItinerarySummary from './itinerary-summary'
 import SimpleRealtimeAnnotation from '../simple-realtime-annotation'
-import { getTotalFareAsString } from '../../../util/state'
 
-const { isBicycle, isTransit } = coreUtils.itinerary
-const { formatDuration, formatTime } = coreUtils.time
+import ItinerarySummary from './itinerary-summary'
+
+const { calculateFares, isBicycle, isTransit, isMicromobility } = coreUtils.itinerary
 
+/**
+ * Obtains the description of an itinerary in the given locale.
+ */
 // FIXME move to core utils
 function getItineraryDescription (itinerary) {
   let primaryTransitDuration = 0
-  let mainMode = 'Walk'
+  let accessModeId = 'walk'
   let transitMode
   itinerary.legs.forEach((leg, i) => {
-    const {duration, mode, rentedBike} = leg
+    const {duration, mode, rentedBike, rentedVehicle} = leg
     if (isTransit(mode) && duration > primaryTransitDuration) {
       // TODO: convert OTP's TRAM mode to the correct wording for Portland
       primaryTransitDuration = duration
-      transitMode = mode.toLowerCase()
+      transitMode = <FormattedMessage id={`common.otpTransitModes.${mode.toLowerCase()}`} />
```

Comment:
Line too long.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/5e2b6708003925b1163418bd629f3870889ec6a2